content/providers/01-ai-sdk-providers/02-openai.mdx

```diff
@@ -804,3 +804,63 @@ const model = openai.image('dall-e-3');
 | ---------- | ------------------------------- |
 | `dall-e-3` | 1024x1024, 1792x1024, 1024x1792 |
 | `dall-e-2` | 256x256, 512x512, 1024x1024     |
+
+## Transcription Models
+
+You can create models that call the [OpenAI transcription API](https://platform.openai.com/docs/api-reference/audio/transcribe)
+using the `.transcription()` factory method.
+
+The first argument is the model id e.g. `whisper-1`.
+
+```ts
+const model = openai.transcription('whisper-1');
+```
+
+OpenAI transcription models support a single `audio` argument which can be a `File`, `Blob`, `Uint8Array`, `ArrayBuffer`, or a `string` (a URL or base64 encoded audio data).
+
+```ts
+const model = openai.transcription('whisper-1', {
+  audio: new File([], 'audio.mp3', { type: 'audio/mp3' }),
+});
+```
+
+You can also pass additional provider-specific options using the `providerOptions` argument.
+
+```ts
+const model = openai.transcription('whisper-1', {
+  providerOptions: { openai: { timestamp_granularities: ['word'] } },
+});
+```
+
+The following provider options are available:
+
+- **timestamp_granularities** _string[]_
+  The granularity of the timestamps in the transcription.
+  Defaults to `['segment']`.
+  Possible values are `['word']`, `['segment']`, and `['word', 'segment']`.
+  Note: There is no additional latency for segment timestamps, but generating word timestamps incurs additional latency.
+
+- **language** _string_
+  The language of the input audio. Supplying the input language in ISO-639-1 format (e.g. 'en') will improve accuracy and latency.
+  Optional.
+
+- **prompt** _string_
+  An optional text to guide the model's style or continue a previous audio segment. The prompt should match the audio language.
+  Optional.
+
+- **temperature** _number_
+  The sampling temperature, between 0 and 1. Higher values like 0.8 will make the output more random, while lower values like 0.2 will make it more focused and deterministic. If set to 0, the model will use log probability to automatically increase the temperature until certain thresholds are hit.
+  Defaults to 0.
+  Optional.
+
+- **include** _string[]_
+  Additional information to include in the transcription response. `logprobs` will return the log probabilities of the tokens in the response to understand the model's confidence in the transcription. `logprobs` only works with `response_format` set to `json` and only with the models `gpt-4o-transcribe` and `gpt-4o-mini-transcribe`.
+  Optional.
```

Comment:
how are the logprobs returned? should we support this particular setting for now?